.github/ISSUE_TEMPLATE/3-bug-report-hip.yml

```
@@ -0,0 +1,42 @@
+name: HIP-specific bug report
+description: Please use this template in case of HIP-specific issue
+labels: ["bug", "hip"]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: how-to-reproduce
+    attributes:
+      label: To reproduce
+      description: Please describe the steps to reproduce the behavior
+      placeholder: |
+        1. Include code snippet as short as possible
+        2. Specify the command which should be used to compile the program
+        3. Specify the comment which should be used to launch the program
+        4. Indicate what is wrong and what was expected
+    validations:
+      required: false
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: Please complete the following information
+      placeholder: |
+        - OS: [e.g Windows/Linux]
+        - Target device and vendor: [e.g. Intel GPU]
```

Maybe specify the example for this template?
```suggestion
        - Target device and vendor: [e.g. AMD GPU]
```